test/disabled/test-sendfd.js

```diff
@@ -50,21 +50,20 @@
 const common = require('../common');
 const assert = require('assert');
 
-const buffer = require('buffer');
 const child_process = require('child_process');
 const fs = require('fs');
 const net = require('net');
 var netBinding = process.binding('net');
 const path = require('path');
 
 var DATA = {
-  'ppid' : process.pid,
-  'ord' : 0
+  'ppid': process.pid,
```

Comment:
This change is unrelated. You may do it in this PR if you want, but you need to reframe the commit message and the PR title differently then.